WMF Framework/CollectionViewCellActionsView.swift

```diff
@@ -147,15 +151,15 @@ public class ActionsView: SizeThatFitsView, Themeable {
 
     public weak var delegate: ActionDelegate?
     
-    @objc func didPerformAction(_ sender: UIButton) {
+    @objc func shouldPerformAction(_ sender: UIButton) {
```

Comment:
A method starting with `should` implies that a `BOOL` is returned that determines whether or not the action is performed. It seems the return value is ignored here.